ocs_ci/deployment/deployment.py

```diff
@@ -359,6 +360,8 @@ def post_ocp_deploy(self):
             enable_huge_pages()
         if config.DEPLOYMENT.get("dummy_zone_node_labels"):
             create_dummy_zone_labels()
+        validation_ui_obj = ValidationUI()
```

Comment:
Why do we have this function call here in `post_ocp_deploy`?

Reply:
this was wrongly added, created a new function `post_ocs_deployment_with_ui` and called it in `deployment_with_ui`